### DISCUSSION TITLE
Get absolute width of paper with percentage width

### DISCUSSION THREAD

**Discussion by TobiasRama** [clientIO/joint#1976] [Q&A]
I initialized my paper with '100%' width and '100%' height, but I need the absolute value in another function. If I try to get it via paper.options.width it returns the string '100%'. I also tried to get the width by paper.$el.width() but it seems to be a little smaller than the actual width. Is there a way to get the absolute width of the paper?

**Comment by jamesgeorgewilliams** [comment]
I think that should be the correct value.

You can use `getComputedSize` too.
https://resources.jointjs.com/docs/jointjs/v3.6/joint.html#dia.Paper.prototype.getComputedSize

<img width="303" alt="Screenshot 2023-01-06 at 17 49 19" src="https://user-images.githubusercontent.com/42288565/211058975-6ab66ce1-f90e-4e00-9427-bab0bfc262d7.png">

If you are taking a reading from tooltips in devTools, they include the border too. This paper has a 1px border, so `getComputedSize` returns `682` which would be correct.
